pat-static/src/main/java/com/rectanglel/patstatic/model/RetrofitPatApi.kt

```diff
@@ -19,50 +17,50 @@
  * @author Jeremy Jao
  * @since 46
  */
-public interface RetrofitPatApi {
+interface RetrofitPatApi {
```

Comment:
I don't understand why the other files were considered delete/adds when this is a "file move"... Would make my life easier if it was always like this :3

Reply:
there's a few like that, probably the diff algorithm didn't give up on them